source/bayernfahrplan/app.d

```diff
@@ -65,18 +66,28 @@ void main(string[] args)
          "ptOptionActive" : "1",
          "mergeDep" : "1",
          "limit" : "20",
-         "deleteAssignedStops_dm" : "1"]);
+         "deleteAssignedStops_dm" : "1"]).to!string.parseJSON;
     // dfmt on
 
     if (substitutionFileName.exists && substitutionFileName.isFile)
     {
         loadSubstitutionFile(substitutionFileName);
     }
 
-    auto currentTime = Clock.currTime;
+    auto currentTime = DateTime.fromISOExtString(content["now"].str);
     JSONValue j = ["time" : "%02s:%02s".format(currentTime.hour, currentTime.minute)];
-    j.object["departures"] = (cast(string) content.data)
-        .parsedFahrplan(reachabilityThreshold).array.JSONValue;
+
+    import bayernfahrplan.fahrplanparser.json.jsonparser : parseJsonFahrplan;
```

Comment:
For now, I moved the imports outside the main function.

In my experience, extracting the commandline parameters handling alone takes quite some time, because one needs to either create lots of module-level variables to store data in, or even define a struct/tuple capable of holding all those.

Given the already HUGE scope of this PR, this should be done elsewhere.